proto/info/info.proto

```diff
@@ -1040,8 +1041,21 @@ message ZInfoNetworkInstance {
   // OBSOLETE - No longer used.
   ZInfoSW softwareList = 9;    // E.g., VPN version; XXX to be deprecated?
 
-  string CurrentUplinkIntf = 10;  // current uplink interface in use by this NI
-  string CurrentUplinkAlias = 11; // user defined namestring alias to this CurrentUplinkIntf, for future
+  // Current uplink interface in use by this NI.
+  // Deprecated - network instance can be associated with multiple device ports,
+  // not just one. To find out which ports are being used by network instance,
+  // read the reported field "ports" instead (see below). Additionally, "routes"
+  // informs about the routes configured for the NI and which ports are currently
+  // used to route the traffic out (this includes info for the default route).
+  string CurrentUplinkIntf = 10 [deprecated = true];
```

Comment:
I saw an example in https://stackoverflow.com/questions/60490487/whats-the-best-way-to-deprecate-a-field-in-protocol-buffer-v3-reserved-vs-depre but not a description of the syntax.

Reply:
This requires some new imports and protoc arguments which I would opt to avoid. I think that having the comment explaining the deprecation is good enough.